controller/db/router_store.go

```diff
@@ -36,23 +36,23 @@ type Router struct {
 	Fingerprint string
 }
 
-func (router *Router) GetId() string {
-	return router.Id
+func (entity *Router) GetId() string {
```

Comment:
`entity`? FWIW, I've been using `self` lately for receivers, so that I don't have to think of a good name for them.

Reply:
I'm guessing `Router` is an "entity", and you're using `entity` across all of the receivers of this type.

Reply:
Exactly. I've been trying to do this specifically in places where there's a lot of copy and paste code, like database entities.